core/src/main/java/org/javarosa/core/model/User.java

```diff
@@ -102,8 +102,8 @@ public String getUsername() {
         return username;
     }
 
-    public String getPassword() {
-        return password;
+    public String getPasswordHash() {
```

Comment:
@amstone326 I think we use this in the CommCare j2me builds as well. Worth taking a look to see if it needs to change outside of /core/ (which you already took a look at)